velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -538,6 +606,119 @@ TEST_P(PrestoSerializerTest, emptyArrayOfRowVector) {
   testRoundTrip(arrayOfRow);
 }
 
+struct SerializeCase {
+  int32_t nullPct;
+  int32_t numSelected;
+  int32_t bits;
+  int32_t vectorSize;
+  uint64_t irTime{0};
+  uint64_t rrTime{0};
+
+  std::string toString() {
+    return fmt::format(
+        "{} of {} {} bit {}%null: {} ir / {} rr",
+        numSelected,
+        vectorSize,
+        bits,
+        nullPct,
+        irTime,
+        rrTime);
+  }
+};
+
+TEST_P(PrestoSerializerTest, timeFlat) {
```

Comment:
We probably should split this into its own benchmark file, there is no validation or dependency on the rest of the file.